tests/project/integration/app/setup.sql

```diff
@@ -0,0 +1,19 @@
+create application role app_public;
+create or alter versioned schema core;
+
+    create or replace procedure core.echo(inp varchar)
```

Comment:
is the indentation here intentional?

Reply:
Yep, it's my preferred way of making the different sections in a large setup script easy to read (each schema is a section).